charts/apisix-ingress-controller/templates/service-apisix.yaml

```diff
@@ -17,4 +21,16 @@ spec:
       targetPort: 9443
   selector:
     {{- include "apisix-ingress-controller.selectorLabels" . | nindent 4 }}
-  type: NodePort
+  type: {{ .Values.gateway.type }}
+  externalTrafficPolicy: {{ .Values.gateway.externalTrafficPolicy }}
+  {{- if eq .Values.gateway.type "LoadBalancer" }}
+  {{- if .Values.gateway.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.gateway.loadBalancerIP }}
+  {{- end }}
+  {{- if .Values.gateway.loadBalancerSourceRanges }}
```

Comment:
it would be better to add some comments for these fields.

Reply:
updated comment.

Reply:
Sorry for my unclear comment. I mean these fields should be in `values.yaml`, like `gateway.annotations`.

Reply:
ohh my bad. Updated values files as suggested